packages/cli/src/util/alias/find-alias-by-alias-or-id.ts

```diff
@@ -1,21 +1,15 @@
 import { Output } from '../output';
 import { Alias } from '../../types';
+
 import Client from '../client';
+import getSafeAlias from '../../util/alias/get-safe-alias';
 
 export default async function findAliasByAliasOrId(
   output: Output,
   client: Client,
   aliasOrId: string
 ) {
-  return client.fetch<Alias | null>(
+  return client.fetch<Alias>(
     `/now/aliases/${encodeURIComponent(getSafeAlias(aliasOrId))}`
   );
 }
-
-function getSafeAlias(alias: string) {
-  return alias
-    .replace(/^https:\/\//i, '')
-    .replace(/^\.+/, '')
-    .replace(/\.+$/, '')
-    .toLowerCase();
-}
```

Comment:
Any reason why this function was moved? It seems like this function is only used in this file.

Reply:
The intention was reduced file size. Making more modular.

Reply:
Let's revert it back to how it was

Reply:
Ok 👍